flink-python/pyflink/fn_execution/beam/beam_coders.py

```diff
@@ -97,6 +97,80 @@ def __hash__(self):
         return hash(self._table_function_row_coder)
 
 
+class BeamAggregateFunctionInputRowCoder(FastCoder):
+    """
+    Coder for Table Function Row.
+    """
+
+    def __init__(self, aggregate_function_row_coder):
+        self._aggregate_function_row_coder = aggregate_function_row_coder
+
+    def _create_impl(self):
+        return self._aggregate_function_row_coder.get_impl()
+
+    def get_impl(self):
+        return BeamCoderImpl(self._create_impl())
+
+    def to_type_hint(self):
+        return typehints.List
+
+    @Coder.register_urn(coders.FLINK_AGGREGATE_FUNCTION_INPUT_SCHEMA_CODER_URN,
+                        flink_fn_execution_pb2.Schema)
+    def _pickle_from_runner_api_parameter(schema_proto, unused_components, unused_context):
+        return BeamAggregateFunctionInputRowCoder(
+            coders.AggregateFunctionInputRowCoder.from_schema_proto(schema_proto))
+
+    def __repr__(self):
+        return 'BeamAggregateFunctionInputRowCoder[%s]' % repr(self._aggregate_function_row_coder)
+
+    def __eq__(self, other):
+        return (self.__class__ == other.__class__
+                and self._aggregate_function_row_coder == other._table_function_row_coder)
+
+    def __ne__(self, other):
+        return not self == other
+
+    def __hash__(self):
+        return hash(self._aggregate_function_row_coder)
+
+
+class BeamAggregateFunctionOutputRowCoder(FastCoder):
```

Comment:
Same to `AggregateFunctionOutputRowCoder`, we don't need `BeamAggregateFunctionOutputRowCoder`